### ISSUE TITLE
`onAdminSave` original page has empty header

### ISSUE THREAD

**Issue by Karmalakas** [getgrav/grav#3259]
In plugin's `onAdminSave` i need a header version of a page before save. `dump($page->getOriginal()->header())` gives me:
```
^ Grav\Common\Page\Header {#566 ▼
  #items: []
  #nestedSeparator: "."
}
```

`toArray()` returns `[]`. Trying to get items also ends up empty.

Also noticed, that `$page->getOriginal()` all flexDirectory objects also have empty headers.
Tried to comment out two lines where `unset($data['header']);` happens in _system/src/Grav/Common/Flex/Types/Pages/PageObject.php:229_, but no luck

Frontmatter before save looks like:
```yml
title: 'Title'
body_classes: 'title-center title-h1h2'
content:
    items: '@self.modular'
visible: false
aura:
    metadata: {}
    pagetype: website
    description: 'Description'
metadata:
    'og:url': 'http://playground.local/test/'
    'og:type': website
    'article:author': 'Author'
```

Might be related to https://github.com/getgrav/grav/issues/3098

**Comment by mahagr** [comment]
Needs also https://github.com/trilbymedia/grav-plugin-flex-objects/commit/19ac5086c106edea47e21ecc1ccfcbb76f826ebc

**Comment by Karmalakas** [comment]
> Needs also [trilbymedia/grav-plugin-flex-objects@19ac508](https://github.com/trilbymedia/grav-plugin-flex-objects/commit/19ac5086c106edea47e21ecc1ccfcbb76f826ebc)

I'm a bit confused :) Did the fix get to v1.0.3 of Flex Objects which was released 02/17/2021? Or will it be released with v1.0.4 after all?

**Comment by rhukster** [comment]
released both
